CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,10 @@
+# Code of Conduct
+
+We follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+<!-- TODO: Decide who will handle Code of Conduct reports and replace [INSERT EMAIL ADDRESS]
```

Comment:
```suggestion
```

Do we want to remove this or keep it for future reference? I just assumed it could be deleted since the contact was added below

Reply:
Thanks for the question. The content was copied from the opentofu main repo, I decided not to assume anything to delete the comment.